src/Sylius/Behat/Context/Ui/Shop/AccountContext.php

```diff
@@ -432,6 +441,22 @@ public function iShouldBeAbleToChangePaymentMethodForThisOrder(OrderInterface $o
     }
 
     /**
+     * @Then I should be redirected to my account dashboard
+     */
+    public function iShouldBeRedirectedToMyAccountDashboard()
+    {
+        Assert::true($this->dashboardPage->isOpen(), 'User should be on the account panel dashboard page but they are not.');
+    }
+
+    /**
+     * @When I want to log in
```

Comment:
I would go with `When I try to log in` and add this step to shop `LoginContext`. Fewer dependencies here and more coherent `LoginContext`.

Reply:
I did that initially but after including the LoginContext in the account suite, I got conflict of step definitions, so I decided to go this way and avoid too many changes. Are we ok with reworking that post release? :)